src/_main_/utils/emailer/send_email.py

```diff
@@ -5,10 +5,14 @@
 import pystmark
 
 from _main_.settings import EMAIL_POSTMARK_SERVER_TOKEN
+from _main_.utils.utils import is_test_mode
 
 FROM_EMAIL = 'no-reply@massenergize.org'
 
 def old_send_massenergize_email(subject, msg, to):
+  if is_test_mode():
+    return True
```

Comment:
when we are in test mode, we should not actually be sending emails.

Reply:
I agree. If we need to test emails we can send to the Postmark test server instead. That should be as simple as swapping the API keys.